try {
  
}
catch (XException exception) {
  
}
catch (YException exception) {
  
}
finally {
	
}

